### DISCUSSION TITLE
大佬，能否开发个obsidian to anki的插件

### DISCUSSION THREAD

**Discussion by shoudeyunkaijianyueming** [guopenghui/obsidian-language-learner#72] [Ideas]
大佬，看你的开发能力真的很强，能不能开发一个对应anki多字段的插件，目前obsidian to anki无法对应anki中多个字段，仅能对应一个字段，而language这个插件单词中英文可以是一个字段，中文可以是一个字段，引用原文可以是另一个字段，您看看能不能开发个插件对应多个字段的，这样可以对单词的导出更有体系（可以一个单词做一个笔记然后阅读中增加的原文自动添加进anki中），同时方便在anki中进一步制卡（比如在anki中增加单词的读音，引用其他词典）【目前来看space可能相对于anki差很多，而language的导出单词功能相对较弱】
而且有个多字段的obsidian to anki 还可以方便其他笔记在anki中的复习
非常希望您能考虑下

**Comment by guopenghui** [comment]
anki的使用我还不太熟悉，后面我研究一下吧。 先转到讨论区

**Comment by vfjy12** [comment]
插件的话，obsidian－to-anki好长时间没有更新了吧，兄弟你要不要试试flashcard，另外我也好想大佬能做一个导入Anki的优化
